package.json

```diff
@@ -24,13 +24,15 @@
   },
   "homepage": "https://github.com/poooi/plugin-quest#readme",
   "dependencies": {
-    "inflection": "^1.12.0"
+    "kcwiki-quest-data": "^0.6.1"
```

Comment:
using `^` will allow installing any versions below 1.0.0, this is dangerous and uncontrollable

Reply:
I'll move to using git submodule instead

Reply:
finally switch to rollup bundling